docker/api/container.py

```diff
@@ -416,12 +416,13 @@ def unpause(self, container):
         res = self._post(url)
         self._raise_for_status(res)
 
-    @utils.minimum_version('1.22')
+    @utils.minimum_version('1.23')
```

Comment:
This should not be changed - people still need to be able to update containers on 1.22